authorization/invitation/service/invitation_service_blackbox_test.go

```diff
@@ -80,6 +85,8 @@ func (s *invitationServiceBlackBoxTest) TestIssueInvitation() {
 			return nil, nil
 		}
 
+		*s.witServiceMock = *test.NewWITMock(t, teamAdmin.IdentityID().String(), spaceName)
```

Comment:
this is a weird syntax. What's the difference with the code below ?
````
s.witServiceMock = test.NewWITMock(t, teamAdmin.IdentityID().String(), spaceName)
````

Reply:
It's still not clear to me: why can't you change the pointer to another value ?

Reply:
let's discuss it in 1 x 1